src/main/java/net/minestom/server/event/inventory/InventoryItemChangeEvent.java

```diff
@@ -58,7 +53,7 @@ public int getSlot() {
     }
 
     @Override
-    public @Nullable Inventory getInventory() {
```

Comment:
I don't think I'm a big fan of the `getInventory` -> `getEventInventory` rename. `getInventory` seems like what someone would assume when looking for that method, and its a breaking change.

Reply:
This was chosen to indicate that it's a breaking change, since otherwise the function would start returning different results when using this PR without a clear indication. I prefer `getInventory` but this was to signify an API change, since it was already a breaking change anyway.

Reply:
This is a largely breaking PR anyway which will probably be merged with a Minecraft update, I would prefer to just accept the break and keep the better method name.